presto-main/src/main/java/com/facebook/presto/execution/SqlTask.java

```diff
@@ -120,6 +125,11 @@ public void stateChanged(TaskState newState)
                     return;
                 }
 
+                // Update failed tasks counter
+                if (newState == TaskState.FAILED) {
```

Comment:
static import `FAILED`

Reply:
TaskState.FAILED and TaskState.ABORTED have already been used in this class without the static import. So I did the same for consistency. Should I change them as well?

Reply:
Yep, please do that in a separate commit (as it's easier to track pure refactors in isolated commits).

Reply:
updated.